sys-cluster/mpich/mpich-3.4.3.ebuild

```diff
@@ -127,6 +127,12 @@ multilib_src_configure() {
 }
 
 multilib_src_test() {
+	# bug #884809
+	# Since valgrind has not been adapted to riscv,
+	# disable it to passed the tests of json-c.
+	if use riscv; then
+		export USE_VALGRIND=0
```

Comment:
Please do this unconditionally -- we usually don't want to run valgrind tests at all because they're flaky on various arches. It's also an automagic test dep here.

Reply:
Strongly agree with disabling valgrind test on all arches, I updated the commit~